src/chrome/content/rules/angband.pl.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+	Invalid certificate:
+		dns1.angband.pl
+		maildns.angband.pl
+		tartarus.angband.pl
+		w2.angband.pl
+
+	Refused:
+		vpn.angband.pl
+
+	Time out:
+		dis1.angband.pl
+		dns0.angband.pl
+		tartarus1.angband.pl
+
+	Unreachable:
+		dis.angband.pl
+		dis2.angband.pl
+		kholdan.angband.pl
+		mordor.angband.pl
+		ntp.angband.pl
+		rhun.angband.pl
+-->
+<ruleset name="angband.pl">
+	<target host="angband.pl" />
+	<target host="www.angband.pl" />
+	<target host="mail.angband.pl" />
+	<target host="pmdk.angband.pl" />
+	<target host="pmdk-dev.angband.pl" />
+
+	<rule from="^http://www\.angband\.pl/" to="https://angband.pl/" />
```

Comment:
This line is superfluous: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects